src/Arcus.Templates.Tests.Integration/Fixture/TestConfig.cs

```diff
@@ -221,12 +222,12 @@ public int GetDockerServiceBusTopicWorkerHealthPort()
         /// </summary>
         /// <param name="entity"></param>
         /// <returns></returns>
-        public string GetServiceBusConnectionString(ServiceBusEntity entity)
+        public string GetServiceBusConnectionString(ServiceBusEntityType entity)
```

Comment:
Same here, should we rename the parameter as well ?

Reply:
Yes, done!